### DISCUSSION TITLE
How Config Datetime to log

### DISCUSSION THREAD

**Discussion by ocarlosapenas** [spatie/laravel-activitylog#768] [Q&A]
Hello, I'm a begginer using activitylog.
I have my model implemented to store some dates, and the dates are saved with correct datetime in DB:

`In model Table:
mydate = 2020-08-31 15:00:00`

But, when a see the logs, the datetime was salved with wrong datetime:
`{
    "old": {
        "mydate": null,
    },
    "attributes": {
        "mydate": "2020-08-31T18:00:00.000000Z",
    }
}`
Adding 3hr;

If I set the cast in model, the log pass to save:
`{
    "old": {
        "mydate": null,
    },
    "attributes": {
        "mydate": "2020-08-31T03:00:00.000000Z",
    }
}`

It's 12hr diff time;
My local time is config to GMT-3 and laravel too.
There is something I'm missing?

**Comment by lyndonjohn** [comment]
I have the same problem too.

**Reply by Gummibeer** [reply]
#822

**Comment by Gummibeer** [comment]
@carlossantoswd if you are in `-03:00` the first two datetimes are the same.
The logged one only contains the offset and is in UTC.
